py_zipkin/logging_helper.py

```diff
@@ -329,6 +338,29 @@ def log_span(
         binary_annotations,
         timestamp_s,
         duration_s,
-    )
-    message = thrift_obj_in_bytes(span)
-    transport_handler(message)
+    ):
+        if not self.transport_handler:
```

Comment:
i believe this check is unnecessary. a logging context is only created if perform_logging is True, which is only True if (self.zipkin_attrs or self.sampling_rate is not None) which requires a check for self.transport_handler.